core/field.js

```diff
@@ -334,24 +326,12 @@ Blockly.Field.prototype.createBorderRect_ = function() {
  * @protected
  */
 Blockly.Field.prototype.createTextElement_ = function() {
-  var xOffset = this.borderRect_ ?
-    this.constants_.FIELD_BORDER_RECT_X_PADDING : 0;
-  var baselineCenter = this.constants_.FIELD_TEXT_BASELINE_CENTER;
-  var baselineY = this.constants_.FIELD_TEXT_BASELINE_Y;
-  this.size_.height = Math.max(this.size_.height, baselineCenter ?
-      this.constants_.FIELD_TEXT_HEIGHT : baselineY);
-  if (this.size_.height > this.constants_.FIELD_TEXT_HEIGHT) {
-    baselineY += (this.size_.height - baselineY) / 2;
-  }
   this.textElement_ = /** @type {!SVGTextElement} **/
       (Blockly.utils.dom.createSvgElement('text',
           {
             'class': 'blocklyText',
-            'y': baselineCenter ? this.size_.height / 2 : baselineY,
-            'dy': this.constants_.FIELD_TEXT_Y_OFFSET,
-            'x': xOffset
           }, this.fieldGroup_));
-  if (baselineCenter) {
+  if (this.constants_.FIELD_TEXT_BASELINE_CENTER) {
```

Comment:
Why is this still here, since you got rid of all other uses of constants?

Reply:
This is a static decision that can be made during init. A renderer decides whether the text is baseline centered and that isn't affected by any theme changes.

Reply:
I see.  So it's back to the same question of how we indicate that some "constants" can be changed by themes, but with a clear reason why this one is static.

Reply:
I've done this here by setting the constant value in the constructor to ``this.DYNAMICALLY_SET`` which it's value is -1 but that doesn't matter as it's value is updated when ``setDynamicConstants`` is called. 
Course, that info isn't easily recognizable for developers.

Reply:
I could also make that clear in the description of a dynamic property. That its real value is set later on.

Reply:
I like having that information in the description, since that's what's going to be exposed by generated jsdocs.

Reply:
Done.